modules.d/01systemd-modules-load/module-setup.sh

```diff
@@ -16,33 +16,30 @@ check() {
 # Module dependency requirements.
 depends() {
 
-    # This module has external dependency on the systemd module.
-    echo systemd
+    # This module has external dependency on other module(s).
     # Return 0 to include the dependent systemd module in the initramfs.
```

Comment:
remove the comment as well

Reply:
I rephrased it to ` # Return 0 to include the dependent module(s) in the initramfs.`  since we should explain why we have a return 0 in the depends() section.

It should be a general rule of thumb that we leave a comment like this since it should be possible for someone else to learn how to use the program or to use a function in a library by reading the comments (and self-help, if provided) without reading the code.